_includes/footer.html

```diff
@@ -22,10 +22,10 @@ <h2 class="footer-heading">
       </div>
       <div class="footer-col footer-col-3">
         <ul>
-          <li><a href="/bylaws">Bylaws</a></li>
-          <li><a href="/membership/agreement">Membership Agreement</a></li>
-          <li><a href="/code-of-conduct">Code of Conduct</a></li>
-          <li><a href="/change-order">Submit a Change Order</a></li>
+          <!-- <li><a href="/bylaws">Bylaws</a></li> -->
+          <li><a href="/membership/agreement">Operating Agreement</a></li>
+          <!-- <li><a href="/code-of-conduct">Code of Conduct</a></li> -->
+          <!-- <li><a href="/change-order">Submit a Change Order</a></li> -->
```

Comment:
At this point it might be cleaner and easier to manage if we build the footer menu like the nav manu. We could add a new `footer: true|false` to each .md page that determines whether that page appears in that menu or not, and manage the pages instead of commenting/uncommenting a menu manually.